app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/204614657-33408eb2-58b7-449c-a8b0-10f9ac619501.png)

- [ ] Your posts controller is not working properly because it is showing an error shown in the image above :arrow_up: Kindly fix this error :blush: